src/app/page.js

```diff
@@ -4,6 +4,7 @@ import dynamic from "next/dynamic";
 import { useEffect, useState } from "react";
 import { api } from "../utils/api";
 import Navigation from "@/components/Navigation";
+import ResearchNavigation from "@/components/researcher/Navigation";
```

Comment:
Remove this! We don't want the researcher's nav bar component on the welcome page.